rali/README.md

```diff
@@ -63,14 +71,32 @@ cmake -DCMAKE_INSTALL_PREFIX=/usr \
       -DCMAKE_INSTALL_DOCDIR=/usr/share/doc/libjpeg-turbo-2.0.3 \
       -DCMAKE_INSTALL_DEFAULT_LIBDIR=lib  \
       ..
-make -j 4
+make -j$nproc
 sudo make install      
 ```
 
-##Jsoncpp installation
+## Jsoncpp installation
 
 sudo apt-get install libjsoncpp-dev
 
+## Google Protobuf installation
+https://github.com/protocolbuffers/protobuf/blob/master/src/README.md
```

Comment:
Currently MIVisionX installs [protobuf v3.5.2](https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX#tested-configurations)

Reply:
Okay. We will remove them

Reply:
The pb files we use from tensorflow requires protobuf version 3.11.4